patches/components-permissions-BUILD.gn.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
+index 799755985b575a7464897d0969bf383c386d267c..80a7190f344b27a58c8c05b710a1b19b292031eb 100644
+--- a/components/permissions/BUILD.gn
++++ b/components/permissions/BUILD.gn
+@@ -2,6 +2,8 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//brave/components/permissions/sources.gni")
```

Comment:
this should go to build/config/brave_build.gni

Reply:
tried that, it doesn't work in this case, chromium file doesn't include any build config.
```
ERROR at //components/permissions/BUILD.gn:92:14: Undefined identifier
  sources += brave_components_permissions_sources
```